apps/server/prettier.config.js

```
@@ -0,0 +1,19 @@
+module.exports = {
```

ESLint

[error] 'module' is not defined.

(no-undef)